data.json

```diff
@@ -125,6 +125,15 @@
     "dateAdded": "2019-10-23T00:39:55.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Deque",
+    "difficulty": "easy",
+    "description": "Fill out the sign up form with your name, email, job title, and company name. Then, respond to Deque's feedback survey",
```

Comment:
No need to mention all the fields to fill, and the description misses the outcome. Answer the survey, and then?
Also, I think it's not that fair to push user to sign up and answer directly. 

Answer Deque's feedback survey once you're logged in to get an awesome t-shirt.